index.html

```diff
@@ -324,13 +324,14 @@
           <dl>
             <dt>ffmpeg</dt><dd>starts the command</dd>
             <dt>-i <i>input_file</i></dt><dd>path and name of the input file</dd>
-            <dt>-write_id3v1 <i>1</i></dt><dd>Write ID3v1 tag. This will add metadata to the old MP3 format, assuming you’ve embedded metadata into the WAV file.</dd>
-            <dt>-id3v2_version <i>3</i></dt><dd>Write ID3v2 tag. This will add metadata to a newer MP3 format, assuming you’ve embedded metadata into the WAV file.</dd>
-            <dt>-dither_method <i>modified_e_weighted</i></dt><dd>Dither makes sure you don’t unnecessarily truncate the dynamic range of your audio.</dd>
+            <dt>-write_id3v1 <i>1</i></dt><dd>Write ID3v1 tag. This will add metadata to the "older" MP3 metadata format that's found at the head of the file, assuming you’ve embedded metadata into the WAV file.</dd>
```

Comment:
This wording is a little unclear. How about something like, "This will create an ID3v1 tag (the older style of MP3 metadata tag, located at the head of the file), and write to it any metadata embedded in the WAV file." ?  
And also adjust the ID3v2 note accordingly